packer/install-dependencies.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash

```

Comment:
@seanknox @jackfrancis: proposal is to replace all the scripts in the packer vhd json with this + the cleanup-vhd script. The script still needs work to handle versions and disabling/removing certain extra things in CSE (eg. etcd on agents) but this is to give you an idea. Main benefit is that we have one source of truth when it comes to what gets installed on clusters so we can evolve the code in one place.